formula-android/src/main/java/com/instacart/formula/fragment/FragmentFlowStore.kt

```diff
@@ -94,18 +96,16 @@ class FragmentFlowStore(
                         }
                         is LifecycleEvent.Added -> {
                             if (!state.activeKeys.contains(key)) {
-                                // TODO: should use input.onScreenError() instead of adding fake render model!
-                                // We want to emit an empty state update if key is not handled.
-                                val notHandled = if (!root.binds(key)) {
-                                    listOf(Pair(key, KeyState(key, "missing-registration")))
+                                if (root.binds(key)) {
+                                    val updated = state.copy(activeKeys = state.activeKeys.plus(key))
+                                    transition(updated)
                                 } else {
-                                    emptyList()
+                                    val updated = state.copy(
+                                        activeKeys = state.activeKeys.plus(key),
+                                        features = state.features.plus(key to FeatureEvent.MissingBinding(key))
```

Comment:
We will delay this error until `FormulaFragment.onCreateView` is called and trigger a crash there.